src/services/mockData.ts

```diff
@@ -0,0 +1,51 @@
+import Notification, { NotificationState } from '../types/Notification'
+import PatientRecord from '../types/PatientRecord'
+
+const notifications: Notification[] = [
+  {
+    id: '123',
+    type: 'Threshold',
+    created: 1605112212,
+    names: 'Bob Smith',
+    patient_id: '0x5a',
+    state: NotificationState.New,
+    golden_id: '0x45',
+    score: 0.5
+  }
+]
+
+const patientRecord: PatientRecord = {
+  auxId: '0x66',
+  type: 'Current',
+  score: 0,
+  uid: '0x98',
+  nationalId: '123456',
+  givenName: 'Bob',
+  familyName: 'Smith',
+  gender: 'Male',
+  dob: new Date(1605522215666),
+  phoneNumber: '085145855',
+  city: 'Welkom',
+  updatedBy: undefined
+}
+
+const goldenRecord: PatientRecord = {
+  auxId: '0x77',
+  type: 'Current',
```

Comment:
```suggestion
  type: 'Golden',
```
right ?

Reply:
This one should actually be empty as it doesn't come from the backend, it's calculated on the front end. All that logic needs to move to the notification endpoint though.